doc: added documentation of CLI Oauth (#1072)
keptn/keptn.github.io: content/docs/0.13.x/operate/user_management/openid_authentication/index.md

weight: 10
---

Please follow the following links for instructions how to setup user authentication using OIDC:

* [Keptn Bridge](../../../reference/bridge/oauth/#enable-disable-authentication)

> * [Keptn CLI](../../../reference/cli/commands/keptn_auth)

Reviewer: This link does not work

Author: in the netlify preview they seem to work

Reviewer: it just gives me github 404 site when I click on it here in github, but when adding ".md" to the end, it works

Reviewer: try here https://github.com/keptn/keptn.github.io/blob/e5ccbe837b4f3fe2e06cf7f1f813944c5fe546c7/content/docs/0.13.x/operate/user_management/openid_authentication/index.md

Author: you need to try here: https://deploy-preview-1072--keptn.netlify.app/docs/0.13.x/operate/user_management/openid_authentication/

Reviewer 2: These Markdown files are compiled into a website using [Hugo](https://gohugo.io/). That's why your link doesn't work @odubajDT  :)

weight: 10
---

> Please follow the following links for instructions how to setup user authentication using OIDC:

Reviewer 2: ```suggestion
Please, follow these links for instructions on how to set up user authentication using OIDC:
```
